mnd/mnd/models.py

```diff
@@ -52,26 +52,15 @@ class PatientInsurance(models.Model):
 
 class PrimaryCarer(models.Model):
 
-    PRIMARY_CARER_RELATIONS = [
-        ('', _("Primary carer relationship")),
-        ('spouse', _("Spouse")),
-        ('child', _("Child")),
-        ('sibling', _("Sibling")),
-        ('friend', _("Friend")),
-        ('other', _("Other(specify)")),
-    ]
-
     LANGUAGE_CHOICES = [
         (l.alpha_2, l.name) for l in pycountry.languages if hasattr(l, 'alpha_2')
     ]
 
-    patient = models.OneToOneField(Patient, related_name='primary_carer', on_delete=models.CASCADE)
+    patients = models.ManyToManyField(Patient, related_name='primary_carers')
```

Comment:
According to the email from Catherine each patient should have just one primary carer, isn't it?
Then why Many to Many?

Reply:
Indeed. This is Many To Many as I didn't want to update the Patient model in TRRF to add a foreign key to this as TRRF does not know about MND specific changes.